server/src/main/java/org/elasticsearch/node/InternalSettingsPreparer.java

```diff
@@ -54,7 +55,19 @@ public static Environment prepareEnvironment(
         loadOverrides(output, properties);
         output.put(input);
         replaceForcedSettings(output);
-        output.replacePropertyPlaceholders();
+        try {
+            output.replacePropertyPlaceholders();
+        } catch (PropertyPlaceholderException e) {
```

Comment:
I think it'd be fine to just catch any exception from this method, no need to be so specific.